example/.storybook/preview.js

```diff
@@ -4,9 +4,12 @@ import { withPaddings } from 'storybook-addon-paddings';
 addDecorator(withPaddings);
 
 addParameters({
-  paddings: [
-    { name: 'Small', value: '16px' },
-    { name: 'Medium', value: '32px', default: true },
-    { name: 'Large', value: '64px' },
-  ],
+  paddings: {
```

Comment:
hey @rbardini, just opening a space for conversation here on how the API looks like. Please let me know your thoughts. :)